build.gradle

```diff
@@ -22,7 +22,7 @@ buildscript {
         gradleVersion = '3.5'
         gradleVersionsPluginVersion = '0.15.0'
         javaVersion = '1.7'
-        katzVersion = '0.1-SNAPSHOT'
+        kategoryVersion = '0.1-SNAPSHOT'
```

Comment:
sorry, I was thinking to extract version to gradle.properties as many maven exported projects do, but that's probably subject for another PR ;)